### DISCUSSION TITLE
Assistance?

### DISCUSSION THREAD

**Discussion by danner26** [Eldritch-Oliver/file-hider#43] [General]
Hello, I see there really hasn't been any work since earlier in 2023. I am currently a maintainer on the NetBox project, and I would be happy to assist with the development of this plugin if you are interested in the help. I just don't want to start working on pull requests if you are not interested. Let me know either way, thanks!

**Comment by Eldritch-Oliver** [comment]
Hello! I'm more than happy to review pull requests and manage releases, I just haven't had the mind to work on this project code-wise. If there is stuff you'd like to see the plugin do/be fixed, feel free to submit a PR following the code style (found in the [contributing guidelines](https://github.com/Oliver-Akins/file-hider/blob/main/.github/contributing.md#code-style))
